### DISCUSSION TITLE
주제선정

### DISCUSSION THREAD

**Discussion by hye-jj** [O-per/cakd3_Project3#3] [Meeting Minutes]
### 쓰레기 분류 : 이미지, 영상
뉴스, 기타 조사
쓰래기 분리배출 어플 : https://play.google.com/store/apps/details?id=kr.or.kprc.recycle&hl=ko&gl=US - 이미지 인식은 아님. qna도 답변빠르고 검색도 됨. 분리배출 설명은 잘 되어 있음.
수퍼빈 :
http://www.aitimes.com/news/articleView.html?idxno=138981 - 재활용품을 기계에 넣으면 분리수거를 해주고 돈을 지급해주는 기계, AI로 이미지를 인식하지만 휴대용으로 개발되지는 않음. 

자료 : 
kaggle_01 : https://www.kaggle.com/aadhavvignesh/pytorch-garbage-classification-95-accuracy
kaggle_02 : https://www.kaggle.com/bouweceunen/garbage-detection-with-tensorflow
자연보존을 위한 쓰래기 집단 데이터 (kaggle): https://www.kaggle.com/brsdincer/garbage-collective-data-for-nature-conservation
**생활 폐기물 이미지(AI허브)**  : 생활 폐기물 탐지 및 분류를 위한 생활 폐기물 15종(가구, 고철류, 나무, 도기류, 비닐, 스티로폼, 유리병류, 의류, 자전거, 전자제품, 종이류, 캔류 등) 이미지 데이터
